trouble running tests on windows OS

the builder tests pass on Travis but not when I run them locally on my trusty windows machine.  It is a weird subprocess error (probably among other things):
```
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

dirname = 'C:\\Users\\simon\\AppData\\Local\\Temp\\regolith_fake'

    def rmtree(dirname):
        """Remove a directory, even if it has read-only files (Windows).
        Git creates read-only files that must be removed on teardown. See
        https://stackoverflow.com/questions/2656322  for more info.

        Parameters
        ----------
        dirname : str
            Directory to be removed
        """
        try:
            shutil.rmtree(dirname)
        except PermissionError:
            if sys.platform == "win32":
>               subprocess.check_call(["del", "/F/S/Q", dirname], shell=True)
E               TypeError: check_call() got an unexpected keyword argument 'she                                             ll'

tests\conftest.py:143: TypeError
_____________________ ERROR at setup of test_builder[cv] ______________________
```
which is weird because according to the docs ``check_call()`` takes shell as an optional arg.

Would love to have this running so I can debug locally, not using Travis!

--- a/news/xonsh_rmtree.rst
+++ b/news/xonsh_rmtree.rst
@@ -1,0 +1,14 @@
+**Added:** None
+
+**Changed:**
+
+* Use ``xonsh.lib.os.rmtree`` in ``conftest.py`` rather that building our own.
+  The xonsh version is expected to do a better job on windows.
+
+**Deprecated:** None
+
+**Removed:** None
+
+**Fixed:** None
+
+**Security:** None

--- a/tests/conftest.py
+++ b/tests/conftest.py
@@ -4,6 +4,7 @@ import json
 import os
 import shutil
 from xonsh.lib import subprocess
+from xonsh.lib.os import rmtree
 import sys
 import tempfile
 from copy import deepcopy
@@ -124,22 +125,3 @@ def make_bad_db():
     yield repo
     os.chdir(cwd)
     rmtree(repo)
-
-
-def rmtree(dirname):
-    """Remove a directory, even if it has read-only files (Windows).
-    Git creates read-only files that must be removed on teardown. See
-    https://stackoverflow.com/questions/2656322  for more info.
-
-    Parameters
-    ----------
-    dirname : str
-        Directory to be removed
-    """
-    try:
-        shutil.rmtree(dirname)
-    except PermissionError:
-        if sys.platform == "win32":
-            subprocess.check_call(["del", "/F/S/Q", dirname], shell=True)
-        else:
-            raise

